lib/pbench/server/api/resources/datasets_compare.py

```diff
@@ -128,21 +128,27 @@ def _get(self, params: ApiParams, req: Request, context: ApiContext) -> Response
         stream_file = {}
         for dataset in datasets:
             try:
-                info = cache_m.get_inventory(dataset.resource_id, "result.csv")
-                file = info["stream"].read().decode("utf-8")
-                info["stream"].close()
+                file = cache_m.get_inventory_bytes(dataset.resource_id, "result.csv")
+            except CacheManagerError as e:
+                raise APIAbort(
+                    HTTPStatus.BAD_REQUEST,
+                    "unable to extract postprocessed data from {dataset.name}",
+                ) from e
             except Exception as e:
                 raise APIInternalError(
-                    f"{dataset.name} is missing 'result.csv' file"
+                    f"Unexpected error extracting postprocessed data from {dataset.name}"
                 ) from e
```

Comment:
Should we be logging anything about `e` here, or do we expect to be able to find it in the log?

Ditto line 73 in `lib/pbench/server/api/resources/datasets_inventory.py`.
Ditto line 77 in `lib/pbench/server/api/resources/datasets_visualize.py`.

Reply:
Raising `APIInternalError` implicitly logs with a stack trace, so no additional log is necessary.